utils/s2n_set.c

```diff
@@ -127,8 +133,7 @@ S2N_RESULT s2n_set_free(struct s2n_set *set)
 
 S2N_RESULT s2n_set_len(struct s2n_set *set, uint32_t *len)
 {
-    ENSURE_REF(set);
-    ENSURE_REF(len);
```

Comment:
why not asserting len?

Reply:
`s2n_array_num_elements` will take care of that already.